docs/modules/spark-k8s/partials/supported-versions.adoc

```diff
@@ -3,5 +3,6 @@
 // Stackable Platform documentation.
 // Please sort the versions in descending order (newest first)
 
-- 3.5.5 (Hadoop 3.3.4, Scala 2.12, Python 3.11, Java 17) (Deprecated)
+- 4.0.0 (Hadoop 3.4.1, Scala 2.13, Python 3.11, Java 17) (Experimental)
```

Comment:
In the description you have a nice list of 3 items *why* it's experimental.
I think this information is gold for users and I would be great to include that list here.